src/Cynthia.Card/src/Cynthia.Card.Server/Locales/pl.json

```diff
@@ -3231,10 +3231,20 @@
       "Info": "Deal 7 damage.",
       "Flavor": "想象一下在最可怕的噩梦中出现的长翅膀的蛇——翼手龙比这更可怕。"
     },
-    "240250": {
-      "Name": "Barbegazi:Promoted",
-      "Info": "Consume an ally and boost self by its power. Resilient.",
```

Comment:
Removing this was a mistake on my part, but I don't believe this code serves any purpose.